ZenPacks/zenoss/Import4/bin/import_tsdb.sh

```diff
@@ -23,9 +23,19 @@ export tsdb_fail_dir="$tsdb_dir/.fail"
 progdir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 source "$progdir/utils.sh"
 
+perf_out ()
+{
+  echo -e "$1" >> "$fail_records"
+}
+
+export -f err_out
```

Comment:
The worker does not have a channel back into the service run instance.. The fail_records file got polled regularly if there is fail import file exists.. Its delta content is then prompted to the service run process.. But, the export -f err_out line is wrong and is not needed.